index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
+Date.prototype.toDateInputValue = (function() {
+  var local = new Date(this);
+  local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+  return local.toJSON().slice(0,10);
+});
+
+document.getElementById('implementationDate').value = new Date().toDateInputValue();
+
+/* Contact Form Validation */
+
+const getForm = document.querySelector('.contact-form');
+
+getForm.addEventListener('submit', function(e){
+  e.preventDefault();
+
+  const items = getForm.querySelectorAll('.cf-item');
+  const formItems = getForm.querySelectorAll('.cf-input');
+
+  const resetForm = function() {
+    for (i = 0; i < items.length; i++) {
+      formItems[i].style.borderColor = '#ccc';
```

Comment:
It's generally better practice to add and remove class names for the particular styling you want rather than modifying the `style` in the JavaScript.